pkgs/by-name/me/mealie/package.nix

```
@@ -28,7 +28,7 @@ let
 
   crfpp = stdenv.mkDerivation {
     pname = "mealie-crfpp";
-    version = "master";
+    version = "unstable-2024-02-12";
```

The date should be the commit date.

```suggestion
    version = "unstable-2021-10-10";
```

---

Ah yes sorry, good catch

---

You accidentally reverted this.